charts/newrelic-infra-operator/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "newrelic-infra-operator.serviceAccountName" . }}
+    namespace: {{ .Release.Namespace }}
+---
+{{/* agent-mock is the ClusterRoleBinding to be used by the ServiceAccoutns of the injected agents */}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}-agent-mock
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "newrelic-infra-operator.fullname" . }}-agent-mock
```

Comment:
creating a comment not to forget to update this :)

Reply:
omw

Reply:
Done!